pages/components/avatar/avatar.html

```diff
@@ -0,0 +1,271 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta
```

Comment:
This is not the right for mate to write meta tags.. The information is passed in contents so.. You cans is :-

<meta name="author" content="Kuldeep">
<meta name="description" content="about you uu">

Reply:
I will correct this. Thanks for the input.